pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -rsxX --reuse-db --tb=native
```

Comment:
Weird how github shows these comments out of order. I copied these from fjord. I don't know if these are the set we prefer. If someone has a set they prefer, I'm all ears. Otherwise I'll figure it out.